streams/src/main/java/org/apache/kafka/streams/state/internals/LeftOrRightValue.java

```diff
@@ -89,8 +74,8 @@ public V2 getRightValue() {
     @Override
     public String toString() {
         return "<"
-            + ((leftValue != null) ? "left," + leftValue : "right," + rightValue)
-            + ">";
+                + ((leftValue != null) ? JoinSide.LEFT + "," + leftValue : JoinSide.RIGHT + "," + rightValue)
```

Comment:
This doesn't seem necessary. LeftOrRightValue can be completely unaware of the JoinSide enum.